audformat/utils/__init__.py

```diff
@@ -13,5 +14,5 @@
     replace_file_extension,
     to_filewise_index,
     to_segmented_index,
-    union,
+    union
```

Comment:
```suggestion
    union,
```

Reply:
Any particular reason why we add comma at the end ? Is it also some standardization ?

Reply:
Yes, if we have multiple arguments / elements etc. over multiple lines we always end with an `,` after the last entry.